sequencer/api/state_signature.toml

```
@@ -0,0 +1,4 @@
+[route.getstatesignature]
+PATH = ["block/:height/statesignature"]
```

Nit-pick, I know this is the style but I'm trying to transition these to being more readable:

```suggestion
[route.get_state_signature]
PATH = ["block/:height/state-signature"]
```

---

Actually since this is going to be in a `state-signature` API module we could maybe just make the route `block/:height`, so that overall it looks like `query.espresso.network/state-signature/block/42`

---

Changed in [f7033f7](https://github.com/EspressoSystems/espresso-sequencer/pull/879/commits/f7033f7222880be8e53eeee798c12ca2691f496c)